cid/helpers/cur.py

```diff
@@ -11,21 +11,42 @@
 
 class CUR(CidBase):
     curRequiredColumns = [
-        'identity_line_item_id',
-        'identity_time_interval',
-        'bill_invoice_id',
-        'bill_billing_entity',
         'bill_bill_type',
-        'bill_payer_account_id',
-        'bill_billing_period_start_date',
+        'bill_billing_entity',
         'bill_billing_period_end_date',
-        'line_item_usage_account_id',
+        'bill_billing_period_start_date',
+        'bill_invoice_id',
+        'bill_payer_account_id',
+        'identity_line_item_id',
+        'identity_time_interval',
+        'line_item_legal_entity',
+        'line_item_line_item_description',
         'line_item_line_item_type',
-        'line_item_usage_start_date',
-        'line_item_usage_end_date',
+        'line_item_operation',
         'line_item_product_code',
+       #'line_item_resource_id',
```

Comment:
Why it's commented?

Reply:
because there was before a separate test on resource id when it was not needed in cudos datasets

Reply:
i put it back but we need to figure out a way to allow customer to install without it